web/src/components/JobVisOptions.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-list-item-content
+    :set="dataset = linkedDataset(pin)"
+  >
+    <v-list-item-title>{{ pinDisplayTitle(pin) }}</v-list-item-title>
+    <v-list-item-subtitle
+      v-for="info in additionalInfo(dataset)"
+      :key="info"
+    >
+      {{ info }}
+    </v-list-item-subtitle>
+    <div
+      v-for="visOption in visOptions(dataset)"
+      v-show="active"
+      :key="visOption.value"
+      :set="showOptions = visualizationsShown.map(vis => vis.value).includes(visOption.value)"
+      @click="(event) => event.stopPropagation()"
```

Comment:
Do you have a specific way to rewrite that line? I tried using click.stop but it didn't appear to stop the propagation, I think the syntax for that action requires that you pass in a function to run instead of propagation.